server/src/main/java/com/gentics/mesh/server/DevRunner.java

```diff
@@ -45,8 +45,6 @@ public static void main(String[] args) throws Exception {
 		defaultOption.getDebugInfoOptions().setLogEnabled(false);
 		OrientDBMeshOptions options = OptionsLoader.createOrloadOptions(OrientDBMeshOptions.class, defaultOption, args);
 
```

Comment:
This is utterly awkward. As a dev, I want the DevRunner to have the ability to start an indexer.